src/dsql/BoolNodes.h

```diff
@@ -138,6 +141,41 @@ class ComparativeBoolNode final : public TypedNode<BoolExprNode, ExprNode::TYPE_
 };
 
 
+class InListBoolNode : public TypedNode<BoolExprNode, ExprNode::TYPE_IN_LIST_BOOL>
+{
+	const static UCHAR blrOp = blr_in_list;
+
+public:
+	InListBoolNode(MemoryPool& pool, ValueExprNode* aArg = nullptr, ValueListNode* aList = nullptr);
+
+	static DmlNode* parse(thread_db* tdbb, MemoryPool& pool, CompilerScratch* csb, const UCHAR blrOp);
+
+	virtual void getChildren(NodeRefsHolder& holder, bool dsql) const
```

Comment:
As it is new code, worth change the `virtual` redeclarations to use `override` instead.

Reply:
Agreed.